compiler/ast.nim

```diff
@@ -271,6 +271,10 @@ type
                       # language; for interfacing with Objective C
     sfDiscardable,    # returned value may be discarded implicitly
     sfOverriden,      # proc is overriden
+    sfCallSideLineinfo# A flag for template symbols to tell the
```

Comment:
Nit, you say this is for template symbols but then you say `the calling side of the macro`.
Just a heads up, the standard term for "call side" is "callsite".

Reply:
yea I will fix that.